analysis/organize_data.R

```diff
@@ -132,18 +116,19 @@ summary(ipumsdata$marriage)
 
 ipumsdata$employment <- factor(ifelse(ipumsdata$empstat==1, "Employed",
                                     ifelse(ipumsdata$empstat==2 |
-                                             ipumsdata$empstat==3,"Unemployed", NA)),
-                             levels=c("Employed","Unemployed"))
+                                             ipumsdata$empstat==3,"Nonemployed", NA)),
+                             levels=c("Employed","Nonemployed"))
 
```

Comment:
The term "unemployed" is misleading because it includes those not in the labor force.